packages/@coorpacademy-components/src/molecule/video-player/jwplayer.js

```diff
@@ -154,6 +164,7 @@ JWPlayer.propTypes = {
           default: PropTypes.boolean
         })
       ),
+      autostart: PropTypes.oneOf(['viewable', 'false']),
```

Comment:
are you sure about it ? why not only boolean ?

Reply:
from https://developer.jwplayer.com/jwplayer/docs/jw8-player-configuration-reference 

```
autostart: String
Whether the player will attempt to begin playback automatically when  a page is loaded. Set to 'viewable' to have player autostart if 50% is  viewable.
```